### DISCUSSION TITLE
Welcome to Advanced-Tab-Groups Discussions!

### DISCUSSION THREAD

**Discussion by 12th-devs** [Vertex-Mods/Advanced-Tab-Groups#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rexxDZN** [comment]
is there a way to have the tab group folder icon filled when collapsed and outlined when not..

**Comment by 12th-devs** [comment]
@rexxDZN Wouldn't be too hard for me to do, give me a bit

**Reply by rexxDZN** [reply]
Actually now that I think about it. A much more loved solution would be folder icons like ARC. At minimum i guess the svg used normally. At best the color part is masked and the color is dynamic to the group color. 
![Folder New](https://github.com/user-attachments/assets/38fd8c5e-6e6e-49da-baec-0b70915db702)
![Icon](https://github.com/user-attachments/assets/b27e3f97-f6cc-44e0-b00e-2c2a6f75cc18)

**Reply by 12th-devs** [reply]
Sadly I cannot do either of these, it might just be me being tired but I tried and failed.

**Reply by 12th-devs** [reply]
@rexxDZN idk if you still need it, but the new version has themed icons per workspace, check them out!

**Comment by PaulMcF1987** [comment]
@Anoms12 I have set this up on 2 computers... Im not sure how the laptop at home works, I done it quickly yesterday
But I have noticed that on my office computer, I need to double click the folders to open/close.
Have I done something wrong or is this the intended way?

**Reply by 12th-devs** [reply]
This seems to be an issue that windows users experience, me included. I have created an issue on Zen but nothing from Maruo yet 😞

**Reply by PaulMcF1987** [reply]
I didnt notice this issue on the other computer although it was a quick set up and go. but my other laptop is on linux o that may be why I never noticed it.
When will this be available as an Addon?
Also, If i add the addon, will that overwrite the current set up or will I need to do anything to remove the current setup?

**Reply by 12th-devs** [reply]
what do you mean as an addon? All of the code in this repo can be used right now.

Edit: If you are talking about the single click, I do not know. I do not know what caused it but it is not the tab group code as it happens to other elements, like the workspace switcher and other user injected elements I have.

**Reply by PaulMcF1987** [reply]
at the top of the page it says `This feature requires manual setup until it becomes available in the Zen Mods store.`
I was meaning if I add it from the Mods store... that way it will automatically update etc

`When will this be available as an Addon?
Also, If i add the addon, will that overwrite the current set up or will I need to do anything to remove the current setup?`

**Reply by PaulMcF1987** [reply]
Also, I have noticed that I cant group pinned tabs, is this correct?

**Reply by 12th-devs** [reply]
Ok, I see what you are saying. I am going to remove that from the readme for 2 reasons

* Marou would never approve it because tab groups are in beta in FF 134

* 135 should be launching in twilight this weekend, meaning tab groups will be enabled so Marou will begin working on them natively 

Hopefully Marou makes it so you can pin them but no, you currently cannot. I appreciate the feedback, and thank you for your time!

**Reply by PaulMcF1987** [reply]
yo might want to add a warning so that people to not try to use this on pinned tabs.

The reason... I tried a few times to see if it would work on pinned tabs but there was no effect. I then removed the CSS and the pinned groupings were ther, however there were lots of groupings, looking kinda messy. I tired to sort it and lost all of my pinned tabs
Also, i dont know what caused it but the browser became quite laggy. In the end I uninstalled and reinstalled Zen to get optimal performance back
I dont think it was your workaround that effected the performance, probably me messing around trying to get the pinned tabs to group

**Comment by clownworld123** [comment]
Hi,thanks for this! But I can not move and reorder groups with my mouse like I can with tabs. Can you make this happen please because I think it is very annoying.

**Reply by 12th-devs** [reply]
I would love to do that too but that is a Firefox issue, I will look into it.
